README.md

```diff
@@ -1,3 +1,36 @@
+# Django Girls blog Petrushynskyi
 [![Build Status](https://travis-ci.org/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii.svg?branch=master)](https://travis-ci.org/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii)
 
-# Django-girls-blog-PetrushynskyiOleksii
+### Installation
+- [Locally](#locally)
+- [Heroku](#heroku)
+
+##### Locally:  
+1. Clone this repository and cd into cloned folder
+2. Install [pyenv](https://github.com/yyuu/pyenv#installation).
+3. Install [pyenv-virtualenv](https://github.com/yyuu/pyenv-virtualenv#installation).
+5. Install Python 3.6.4 : `pyenv install 3.6.4`.
+6. Create a new virtualenv : `pyenv virtualenv 3.6.4 <name of virtualenv>`.
+7. Activate virtualenv : `pyenv local <name of virtualenv>`.
+8. Install requirements for a project : `pip install -r requirements.txt`
+9. Add .env in mysite folder, like in .env.example.
```

Comment:
You should return example file (and list some secret there)

Reply:
done